editor/editor_feature_profile.h

```diff
@@ -49,6 +49,7 @@ class EditorFeatureProfile : public RefCounted {
 	enum Feature {
 		FEATURE_3D,
 		FEATURE_SCRIPT,
+		FEATURE_GAME,
```

Comment:
If you added this at the end (above MAX), you'd avoid changing almost every value in the enum. The order isn't that important.